DetectorDescription/Detectors/src/SimpleCylinder_geo.cxx

```diff
@@ -0,0 +1,53 @@
+// $Id: $
+//====================================================================
+//  AIDA Detector description implementation for FCC
+//-------------------------------------------------------------------- 
+// Simple cylinder using Tube to be used to define cylinder composed of 1 single material
+//
+// @author Clement Helsens
+//====================================================================
+#include "DD4hep/DetFactoryHelper.h"
+
+using namespace std;
+using namespace DD4hep;
+using namespace DD4hep::Geometry;
+
+
+static Ref_t create_element(LCDD& lcdd, xml_h e, SensitiveDetector sens)  {
```

Comment:
Same comments as for `SimpleCone_geo.cxx`: Namespace, name and unused parameter warning.